templates/web/tfl/report/new/roads_message.html

```diff
@@ -1,4 +1,9 @@
 <div id="js-roads-responsibility" class="box-warning hidden">
+    <div id="js-not-tfl-road" class="hidden js-responsibility-message">
```

Comment:
This will need adding to .com roads_message.html if you want it to work there, I think.
Though perhaps not necessary if these categories will only appear once a red route is clicked. But for now...